### DISCUSSION TITLE
Welcome to starter-kit Discussions!

### DISCUSSION THREAD

**Discussion by vinzvinci** [SurPathHub/starter-kit#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to discuss every roadmap for beginners. We hope that you:
  * Create discussions if you want to add new <a href="https://github.com/SurPathHub/starter-kit/tree/main/roadmap">roadmap</a> before making an issues and make a PR to add new <a href="https://github.com/SurPathHub/starter-kit/tree/main/roadmap">roadmap</a> .
  ex:
    - Web Development  
    - Data Scientist
    - DevOps Engineer
    - Etc.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by Cahllagerfeld** [comment]
>Front-end development
Back-end development

I'd probably split the topic you mentioned. Backend for example could be 
- Backend
    - Express.js
    - Flask
    - Django
    - Spring
    - ...
    - 
or even more detailed 
- Backend
    - Nodejs
        - express
        - polka
        - fastify

**Reply by Cahllagerfeld** [reply]
Tbh for me its quite hard to name any ressources you should have a look at when starting with a new framework or library. For me personally it's more like trial and error and trying to solve me problem by using google lol.

**Reply by vinzvinci** [reply]
Yeah lol. Thanks for the suggestion, I will try my best to make it done.

**Comment by Cahllagerfeld** [comment]
You could potentially make a section for IDEs which can be used for the different language in specific scenarios maybe.

**Reply by vinzvinci** [reply]
I'll take note of this.

**Comment by vinzvinci** [comment]
I think crash course from different dev. is enough @Cahllagerfeld. I don't want it to make it big, just a guide for a starter. 

`Just quick documentation for every crash course and video link:`
Example
- Web Dev Crash course by **Traversy Media**
- Full stack crash course 
- Back end crash course
- Front end crash course

**Reply by Bryson69** [reply]
This is an awesome idea. I like it.

**Reply by vinzvinci** [reply]
If you have an idea. It will be cool if some of you start a discussion for those I suggested (separated topic). Feel free to contribute folks.

`ex:`
Title:  **Web Development Crash Course**

**Comment by vinzvinci** [comment]
I've created <a href="https://github.com/SurPathHub/starter-kit/discussions/3">Roadmap content</a> discussion. Please do follow
